src/coreclr/nativeaot/Runtime/eventtrace_bulktype.cpp

```diff
@@ -295,8 +282,13 @@ int BulkTypeEventLogger::LogSingleType(MethodTable * pEEType)
     // Determine this MethodTable's module.
     RuntimeInstance * pRuntimeInstance = GetRuntimeInstance();
 
-    ULONGLONG osModuleHandle = (ULONGLONG) pEEType->GetTypeManagerPtr()->AsTypeManager()->GetOsModuleHandle();
-
+    // There can be invalid pEETypes where EETypeElementType is ElementType_Unknown and accessing the TypeManager can cause problems
```

Comment:
Where do these invalid EETypes come from?

Reply:
I'm not sure.

The symbols for these are like: `__GCStaticEEType_NNN` (where NNN is like 010, 001). 

`ElementType_Unknown` is the only common trait I found to distinguish although `m_usNumVtableSlots`, `m_usNumInterfaces`, `GetHashCode`() tend to be very large (0xfff0, 0xffff, 0xffffffff) but not consistent. 

Using DIA's `findSymbolByRVAEx` gives the following information for these addresses,

```
Displacement = 0x0
                        SymIndex:        0x1
                          SymTag:        0x5
                            Name:        __GCStaticEEType_010
                   LexicalParent:        00000245432CA850
                            Type:        00000245432CA850
                    LocationType:        0x1
                  AddressSection:        0x3
                   AddressOffset:        0xC0D00
          RelativeVirtualAddress:        0x6FBD00
                  VirtualAddress:        0x6FBD00
                          Length:        0x28
                 LexicalParentId:        0x2
                          TypeId:        0x3
                 UndecoratedName:        __GCStaticEEType_010
                        NoReturn:        false
  uses custom calling convention:        false
                        NoInline:        false
   debug info for optimized code:        false
                     Unreachable:        false
           return from interrupt:        false
                      far return:        false
           frame pointer present:        false

Function       : static, [006FBD00][0003:000C0D00], len = 00000028, __GCStaticEEType_010
                 Function attribute:
                 Function info:
FuncDebugStart :   static, [006FBD00][0003:000C0D00]
FuncDebugEnd   :   static, [006FBD28][0003:000C0D28]

```

Reply:
These are EETypes for the GC statics.